cmd/run

```diff
@@ -144,7 +144,12 @@ _run() {
   fi
 
   if [[ -f "${VMRUN}/pidfile" ]]; then
-    _fatal 1 "'${VMRUN}/pidfile' exists; vm might already be running"
+    if [[ "$(pidof $QEMU_SYSTEM_BINARY)" ]]; then 
+      _fatal 1 "'${VMRUN}/pidfile' exists; vm might already be running"
+    else
+      echo "Removing the old '${VMRUN}/pidfile'"
```

Comment:
Use the `_log` helper for log info messages and keep it lower case, e.g. `"removing stale pidfile '${VMRUN}/pidfile'"`

Reply:
`stale` is the word I was looking for!